ansible/roles/condor_launcher/tasks/main.yml

```diff
@@ -1,50 +1,58 @@
 ---
-    - ansible.builtin.shell: "curl -s https://api.github.com/repos/cyverse-de/condor-launcher/releases/latest | jq -r '.tag_name'"
-      register: tag_name
-      delegate_to: 127.0.0.1
-      become: false
-
-    - ansible.builtin.shell: "echo https://github.com/cyverse-de/condor-launcher/releases/download/{{ tag_name.stdout }}/condor-launcher"
-      register: condor_launcher_url
-      delegate_to: 127.0.0.1
-      become: false
-
-    - debug:
-        msg: "{{ condor_launcher_url.stdout }}"
-
-    - ansible.builtin.shell: "echo https://raw.githubusercontent.com/cyverse-de/condor-launcher/{{ tag_name.stdout }}/condor-launcher.service"
-      register: condor_launcher_service_url
-      delegate_to: 127.0.0.1
-      become: false
-    
-    - debug:
-        msg: "{{ condor_launcher_service_url.stdout }}"
-
-    - ansible.builtin.get_url:
-        url: "{{ condor_launcher_url.stdout }}"
-        dest: "/usr/local/bin/condor-launcher"
-        mode: 0755
-        force: true
-
-    - ansible.builtin.get_url:
-        url: "{{ condor_launcher_service_url.stdout }}"
-        dest: "/etc/systemd/system/condor-launcher.service"
-        mode: 0644
-        force: true
-
-    - ansible.builtin.template:
-        src: "{{ de_releases_dir}}/templates/jobservices.yml.j2"
-        dest: /etc/jobservices.yml
-        mode: 0644
-
-    - ansible.builtin.systemd:
-        daemon_reload: yes
-    
-    - ansible.builtin.systemd:
-        name: condor-launcher
-        state: stopped
-        enabled: true
-
-    - ansible.builtin.systemd:
-        name: condor-launcher
-        state: started
+- ansible.builtin.user:
+    user: clauncher
+    create_home: yes
+    append: true
+    groups:
+      - docker
+      - condor
```

Comment:
It occurs to me now that the `clauncher` account also needs to be created on worker nodes. 😁 I'll fix that shortly.